### PR TITLE
Changes to the sample project

### PR DESCRIPTION
**Pull request by ozoromo**
Edited sample-test.js to fix a bad contract tx that threw an error before. (count wasn't json formatted)

Changed the accounts in sample-test.js to be the actual accounts used in the Docker testnet (localnet)

### CODE REVIEW

File: packages/polar/sample-project/polar.config.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+  },
+  {
+    name: 'account_2',
+    address: 'secret1ajz54hz8azwuy34qwy9fkjnfcrvf0dzswy0lqq',
+    mnemonic: 'chair love bleak wonder skirt permit say assist aunt credit roast size obtain minute throw sand usual age smart exact enough room shadow charge'
+  },
+  {
+    name: 'account_3',
+    address: 'secret1ldjxljw7v4vk6zhyduywh04hpj0jdwxsmrlatf',
+    mnemonic: 'word twist toast cloth movie predict advance crumble escape whale sail such angry muffin balcony keen move employ cook valve hurt glimpse breeze brick'
```
Comment (coco-sha) [inline packages/polar/sample-project/polar.config.js:20]: Account information will be different for each instance of local network. Could you please restore this file ?

File: packages/polar/sample-project/test/sample-test.js [hunk 2]
```diff
@@ -21,15 +21,15 @@ describe("sample_project", () => {
...
     const { contract_owner, other, contract } = await setup();
     const deploy_response = await contract.deploy(contract_owner);
-    
+
     const contract_info = await contract.instantiate({"count": 102}, "deploy test", contract_owner);
-    
-    await expect(contract.tx.reset({account: other}, 100)).to.be.revertedWith("unauthorized");
-    await expect(contract.query.get_count()).not.to.respondWith({ 'count': 1000 });
+
+    await expect(contract.tx.reset({account: other}, {"count": 100})).to.be.revertedWith("unauthorized");
```
Comment (coco-sha) [inline packages/polar/sample-project/test/sample-test.js:31]: :+1:

**Review by coco-sha — CHANGES_REQUESTED** [review]
Thanks for creating this PR. I have suggested one small change.

File: packages/polar/sample-project/polar.config.js (hunk 1 above)
Comment (ozoromo) [reply packages/polar/sample-project/polar.config.js]: I changed it back to the default as you asked, but are you sure the accounts are unique to each local test network? 
I vaguely remember extracting the mnemonics directly from the docker images source code.

**Review by ozoromo — COMMENTED** [review]
reviewed changes

**Review by coco-sha — APPROVED** [review]

**Pull request merged**